README.md

```diff
@@ -3297,11 +3297,23 @@ http_basic do |username, password|
 end
 ```
 
-```ruby
-http_digest({ realm: 'Test Api', opaque: 'app secret' }) do |username|
-  # lookup the user's password here
-end
-```
+For Digest Auth
+
+* Compare with the pure password
+
+    ```ruby
+    http_digest({ realm: 'Test Api', opaque: 'app secret' }) do |username|
+      # lookup the user's password here
+    end
+    ```
+
+* If you want to storing the ha1 hash: `MD5(username:realm:password)` and compare with it
```

Comment:
`to storing` -> `to store`
`ha1` -> `sha` 

The example uses MD5 and not a SHA1. I would remove the bullets and just provide the two examples with my suggested text above.